CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+- BREAKING--Convert datastar models to dynastar models
```

Comment:
for now we have to publish this as a new module with -dynamo postfix and publish it as a `npm version premajor` so we can still keep tags in this repo. This PR will be merged to a `dynamo` branch. I can make that for you

Reply:
So should I be doing the versioning stuff on this PR, or later?